docs/src/components/Footer/index.js

```diff
@@ -108,54 +110,62 @@ const LinksContactUs = () => {
     </div>
   );
 };
-const Contributors = () => {
-  const { siteConfig } = useDocusaurusContext();
-  const githubApiContributors = siteConfig?.customFields?.githubApiContributors;
-  const [isLoaded, setIsLoaded] = useState(false);
-  const [items, setItems] = useState([]);
-  //getting the top contributors from github by sending the api order as desc
-  useEffect(() => {
-    fetch(githubApiContributors)
-      .then((res) => res?.json())
-      .then(
-        (result) => {
-          setIsLoaded(true);
-          setItems(result);
-        },
-        (error) => {
-          setIsLoaded(true);
-          console.error(error);
-        }
-      );
-    return () => {
-      setItems([]);
-    };
-  }, []);
+const Contributors = ({ isTopContributors }) => {
   return (
     <div className="col">
-      {isLoaded && items.length && (
+      {(
         <>
-          <span className="footer__title">
-            <Translate
-              id="component.Footer.topContributors"
-              description="Top contributors label in footer"
-            >Top contributors</Translate>
-          </span>
-          <ul className="footer__items">
-            {items?.slice(0, 3).map((item) => {
-              return (
-                <li className="footer__item" key={item.login}>
-                  <Link
-                    className="footer__link-item"
-                    to={item.html_url}
-                    target="_blank"
-                  >
-                    {item.login}
-                  </Link>
-                </li>
-              );
-            })}
-          </ul>
+          {
+            isTopContributors ? (
```

Comment:
add respective check for both topcontributors and newcontributors, the new contributors list can come up empty. So better we add the null check with both `topContributors` and `newContributors`